packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -1171,3 +1318,42 @@ abstract class PlatformViewController {
   /// Clears the view's focus on the platform side.
   Future<void> clearFocus();
 }
+
+/// An interface for controlling a platform view whose gestures passed from gesture
+/// recognizer.
+abstract class GestureRecognitionPlatformViewController {
+
+  /// The viewId associated with this controller.
```

Comment:
What is `viewId` a reference to? It looks like code, but isn't part of this class.